lib/rgl/base.rb

```diff
@@ -329,4 +329,22 @@ def each_edge_aux
 
   end # module Graph
 
+  def set_vertex_options(vertex, **options)
+    @vertex_options ||= {}
+    @vertex_options[vertex] ||= {}
+
+    RGL::DOT::NODE_OPTS.each do |opt|
+      @vertex_options[vertex][:"#{opt}"] = options[:"#{opt}"] if options.key?(:"#{opt}")
+    end
+  end
+
+  def set_edge_options(edge, **options)
```

Comment:
In the tests you use the string representation "a-b" of an edge as a key in the _edge_options_ dictionary. I think it would be better to use the an `Edge` as key internally. And change the signature to:

```ruby
def set_edge_options(u, v, **options)
  edge = edge_class.new(u, v)
  ...
end

Reply:
I've swapped this out. Please check if this change makes sense.